langchain4j-redis/src/main/java/dev/langchain4j/store/embedding/redis/DataType.java

```diff
@@ -0,0 +1,23 @@
+package dev.langchain4j.store.embedding.redis;
+
+/**
+ * Redis vector field data type
+ */
+public enum DataType {
```

Comment:
Actually I want to support user customizing  `RedisSchema`. But at this stage I think this function is not very important, so I think it can be retained package-public. It doesn't matter